docs/content/latest/api/ysql/datatypes/type_datetime/operators/test-date-time-multiplication-overloads.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Test the date-time multiplication overloads [YSQL]
+headerTitle: Test the date-time multiplication overloads
+linkTitle: Test multiplication overloads
+description: Presents code that tests the date-time multiplication overloads. [YSQL]
+menu:
+  latest:
+    identifier: test-date-time-multiplication-overloads
+    parent: date-time-operators
+    weight: 40
+isTocNested: true
+showAsideToc: true
+---
+
+qqq
```

Comment:
Again...

Reply:
Ad hoc note to slef to fix/review. Fixed now.